.xe/specs/blueprint/plan.md

```diff
@@ -181,6 +181,234 @@ Features will be implemented in `.xe/specs/{feature-id}/` directories as they ar
 
 ---
 
+## Feature Breakdown
+
+### Phase 1: POC - Early Adopters (11 features)
```

Comment:
There's no need to mention the number of features. That will get outdated. Update this everywhere.